Lib/test/test_pwd.py

```diff
@@ -21,7 +21,8 @@ def test_values(self):
             self.assertEqual(e[3], e.pw_gid)
             self.assertIsInstance(e.pw_gid, int)
             self.assertEqual(e[4], e.pw_gecos)
-            self.assertIsInstance(e.pw_gecos, str)
+            if e.pw_gecos is not None:
+                self.assertIsInstance(e.pw_gecos, str)
```

Comment:
```suggestion
            self.asserIn(type(e.pw_gecos), (str, type(None)))
```

Reply:
Thank you, Victor, for looking at this PR. I had used the `is not None` check because Serhiy had recommended it on the BPO issue.